<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calibrationViewer</class>
 <widget class="QWidget" name="calibrationViewer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration Viewer</string>
  </property>
  <property name="windowIcon">
   <iconset resource="systray.qrc">
    <normaloff>:/images/smith_chart_red.png</normaloff>:/images/smith_chart_red.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QComboBox" name="comboBox">
       <item>
        <property name="text">
         <string>Frequency Calibration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Path Magnitude Calibration</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Path Phase Calibration</string>
        </property>
       </item>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBox">
       <property name="text">
        <string>Limit to the data of interest</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QFrame" name="chartView">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="systray.qrc"/>
 </resources>
 <connections/>
</ui>
